lib/ljglibs/gtk/window.moon

```diff
@@ -83,4 +83,14 @@ core.define 'GtkWindow < GtkBin', {
   set_default_icon_from_file: (filename) ->
     catch_error(C.gtk_window_set_default_icon_from_file, filename) != 0
 
+  save_screenshot: (filename, format='png') =>
+    gdkwindow = ffi.C.gtk_widget_get_window ffi.cast 'GtkWidget*', @
+    shot = ffi.C.gdk_pixbuf_get_from_window gdkwindow, 0, 0, @allocated_width, @allocated_height
+    err = ffi.new 'GError *[1]'
+    ffi.C.gdk_pixbuf_save shot, filename, format, err, nil
```

Comment:
In accordance with the above, this would the `pixbuf\save(shot, filename, format)`. Also note that the error handling is missing a `g_error_free`.